src/kbmod/work_unit.py

```diff
@@ -37,41 +37,85 @@ class WorkUnit:
     wcs : `astropy.wcs.WCS`
         A global WCS for all images in the WorkUnit. Only exists
         if all images have been projected to same pixel space.
+    constituent_images: `list`
+        A list of strings with the original locations of images used
+        to construct the WorkUnit. This is necessary to maintain as metadata
+        because after reprojection we may stitch multiple images into one.
```

Comment:
High level question (that I should have realized and asked in the previous iteration): What happens to the image stack if multiple images are stitched together? Does the size of the image stack change?

Similarly can the size of the `per_image_wcs` and `per_image_ebd_wcs` lists be different?

From the code it looks like the answers are yes for the size of the image stack, but no for the size of the lists. Is this correct?

Reply:
yes that's correct. Since we can stitch multiple images together and these are stored as one `LayeredImage` in the image stack, we keep the `per_image_wcs` and `per_image_ebd_wcs` around, which matches the amount of `constituent_images` in the `WorkUnit`, but not necessarily the image stack. Then when we need to transform coordinates from the reprojected `WorkUnit` back to the original coordinate space, we can

1. get the position in the reprojected image
2. get the index(es) of the constituent image(s) contained in that image
3. `invert_correct_parallax` on the  coordinate
4. determine the pixel positions in the original image from the `per_image_wcs`
5. in the case where a portion of the image stack has been tiled, we can check against all of the constituent wcses and determine which pixel position was actually in bounds

Reply:
How will the backward mapping in step 2 done? Will you need to add list of lists mapping the ebd transformed images back to the corresponding constituent images? Or can that already be done with the current listS?

Reply:
oh right... let me think about this. Yeah we'll need some other way to store which `constituent_images` are in what layer of the `WorkUnit`